ocis-pkg/registry/register.go

```diff
@@ -16,12 +16,12 @@ func RegisterService(ctx context.Context, service *mRegistry.Service, logger log
 
 	logger.Info().Msgf("registering external service %v@%v", node.Id, node.Address)
 
-	rOpts := []mRegistry.RegisterOption{mRegistry.RegisterTTL(time.Minute)}
+	rOpts := []mRegistry.RegisterOption{mRegistry.RegisterTTL(GetRegisterTTL())}
 	if err := registry.Register(service, rOpts...); err != nil {
 		logger.Fatal().Err(err).Msgf("Registration error for external service %v", service.Name)
 	}
 
-	t := time.NewTicker(time.Second * 30)
+	t := time.NewTicker(GetRegisterInterval())
```

Comment:
reva services were registered every 30 sec with a ttl of a minute ...

so we slightly increase the amount of registrations ... but they can be tweaked with the `EXPERIMENTAL_REGISTER_INTERVAL=25` and `EXPERIMENTAL_REGISTER_TTL=30` envl vars.